make-checkout

```diff
@@ -70,11 +70,24 @@ def main():
     execute("git", "checkout", "--detach", opts.revision, cwd=TARGET_DIR, error_on_fail=False)
 
     # get bots/
-    if api.repo != "cockpit-project/cockpit" or (opts.base and opts.base != "master"):
-        execute("git", "remote", "add", "bots_origin", "https://github.com/cockpit-project/cockpit", cwd=TARGET_DIR)
-        execute("git", "fetch", "--depth=1", "bots_origin", opts.bots_ref or "master", cwd=TARGET_DIR)
-        bots_ref = execute("git", "rev-parse", "FETCH_HEAD", cwd=TARGET_DIR).strip()
-        execute("git", "checkout", "--force", bots_ref or "bots_origin/master", "--", "bots/", cwd=TARGET_DIR, error_on_fail=False)
+    bots_clone_dir = os.path.join(TARGET_DIR, "bots")
+
+    # HACK
+    # As long as bots/ is duplicated in cockpit, keep the conditional section here
+    if api.repo == "cockpit-project/cockpit":
+        if opts.base and opts.base != "master":
```

Comment:
At first sigh the original condition was correct -- you also need to run the "else:" parts for non-master cockpit branches, as they also don't have bots/. However, I think this special case should  look differently: If you clone cockpit/master, it should remove its bots/ directory afterwards. This makes this whole code part much easier to understand, and will run through the bots/ setup the same way for every PR.

Reply:
Ack